app/components/recommended-video.js

```diff
@@ -170,7 +170,10 @@ export default Component.extend(NoScrollMixin, {
 	},
 
 	getVideoData() {
-		return fetch(`https://cdn.jwplayer.com/v2/playlists/${this.get('playlistId')}?related_media_id=${this.get('relatedMediaId')}`).then((response) => response.json());
+		/* eslint-disable-next-line max-len */
+		const url = `https://cdn.jwplayer.com/v2/playlists/${this.get('playlistId')}?related_media_id=${this.get('relatedMediaId')}`;
```

Comment:
new lines are ommited in fetch from what I tested today
so you can safely break the line!